docsrc/dynamo/dynamo_export.rst

```diff
@@ -1,22 +1,22 @@
 .. _dynamo_export:
 
-Torch-TensorRT Dynamo Backend
+Torch-TensorRT Dynamo Frontend
```

Comment:
Maybe this should be renamed "AOT Path". Otherwise, the section (currently) looks like:
```
Dynamo Frontend
|
--- Torch-TensorRT torch.compile Backend
--- Torch-TensorRT Dynamo Frontend
```
Additionally, the words "backend" and "frontend" are used often in both articles, so the real meaning becomes more challenging to discern.